homeassistant/components/person/__init__.py

```diff
@@ -187,7 +191,9 @@ def entities_in_person(hass: HomeAssistant, entity_id: str) -> list[str]:
 class PersonStore(Store):
     """Person storage."""
 
-    async def _async_migrate_func(self, old_major_version, old_minor_version, old_data):
+    async def _async_migrate_func(
+        self, old_major_version: int, old_minor_version: int, old_data: dict[str, Any]
+    ) -> dict[str, Any]:
```

Comment:
Does it make sense to use `homeassistant.json.JsonObjectType` here instead?

Reply:
That's possible but it doesn't change much.

At the moment, the `Store` class is only partially typed anyway. Once that changes, we might need to adjust the downstream types slightly anyway to conform with subtyping rules. Adding the basic types here is good to check that the correct types are added when `Store` is fully annotated.